utils_cv/detection/helper.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The name "Helper.py" seems a bit too generic. Maybe bbox.py instead for everything bbox related?